src/lib/tvgPictureImpl.h

```diff
@@ -47,30 +54,45 @@ struct Picture::Impl
 
     void reload()
     {
-        if (loader && !paint) {
-            auto scene = loader->data();
-            if (scene) {
-                paint = scene.release();
-                if (!paint) return;
-                loader->close();
+        if (loader) {
+            if (!paint) {
+                auto scene = loader->root();
+                if (scene) {
+                    paint = scene.release();
+                    if (!paint) return;
+                    loader->close();
+                }
+            }
+            if (!buffer) {
+                buffer = (uint32_t*)loader->data();
```

Comment:
because error occured
```
../src/lib/tvgPictureImpl.h:67:40: error: invalid conversion from ‘const uint32_t*’ {aka ‘const unsigned int*’} to ‘uint32_t*’ {aka ‘unsigned int*’} [-fpermissive]
   67 |                 pixels = loader->pixels();
      |                          ~~~~~~~~~~~~~~^~
      |                                        |
      |                                        const uint32_t* {aka const unsigned int*}
```